build.gradle

```diff
@@ -115,23 +115,28 @@ tasks.withType(JavaCompile) {
 
 test {
   useJUnitPlatform()
-}
-
-// Configures jacoco test coverage to be included when "test" is run
-test {
   finalizedBy jacocoTestReport
-}
-jacocoTestReport {
-  dependsOn test
-}
-// Enabling the XML report allows for sonar to grab coverage data from jacoco
-jacocoTestReport {
-  reports {
-    // This isn't working with Gradle 8. Will replace this soon with the sonar instance in docker-compose.
-    // xml.enabled true
+  jacocoTestReport {
+    dependsOn test
+    reports {
+      xml.required = true
+    }
   }
-}
 
+  // Turning off coverage verification, as we haven't figured out how to configure Sonar correctly to pick up the jacoco
```

Comment:
This comment isn't true - it was relevant for the Spark connector which had a separate code-coverage-report project, but that's not the case here. Should be able to either remove this or set it to a value slightly lower than whatever the current code coverage is.